src/pretix/base/email.py

```diff
@@ -258,9 +258,30 @@ def _placeholder_payment(order, payment):
         return str(payment.payment_provider.order_pending_mail_render(order))
 
 
+def get_best_name(position_or_address, parts=False):
```

Comment:
I think having the `position_or_address` context creation input smells, but I guess it's not possible for a placeholder right now to specify that it requires either a position or an address?